tunnelblick/Tunnelblick.xcodeproj/project.pbxproj

```diff
@@ -2058,6 +2096,7 @@
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VALUE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = "../third_party/build/openssl-1.0.2k/include/";
```

Comment:
some modules require openssl features, so i added build openssl path as headers search path. i'm not sure, is there a better way to link to openssl?